hw/ip/i2c/dv/Makefile

```diff
@@ -30,10 +41,37 @@ include ${DV_DIR}/../../../dv/tools/common_tests.mk
 
 ifeq (${TEST_NAME},i2c_sanity)
   UVM_TEST_SEQ   = i2c_sanity_vseq
+  RUN_OPTS      += +en_scb=0
+endif
+
+ifeq (${TEST_NAME},i2c_rx_tx)
+  UVM_TEST_SEQ   = i2c_rx_tx_vseq
+  RUN_OPTS      += +en_scb=0
+endif
+
+ifeq (${TEST_NAME},i2c_override)
+  UVM_TEST_SEQ   = i2c_override_vseq
+  RUN_OPTS      += +en_scb=0
+endif
+
+ifeq (${TEST_NAME},i2c_reg_access)
+  UVM_TEST_SEQ   = i2c_reg_access_vseq
+  RUN_OPTS      += +en_scb=0
 endif
 
 ####################################################################################################
 ## Include the tool Makefile below                                                                ##
 ## Dont add anything else below it!                                                               ##
 ####################################################################################################
 include ${DV_DIR}/../../../dv/tools/Makefile
+
+#TODO: later remove
+i2c-all:
+	@make TEST_NAME=i2c_override
+	@make TEST_NAME=i2c_reg_access
+
+csr-all:
+	@make TEST_NAME=i2c_csr_hw_reset
+	@make TEST_NAME=i2c_csr_rw
+	@make TEST_NAME=i2c_csr_bit_bash
+	@make TEST_NAME=i2c_csr_aliasing
```

Comment:
My suggestion is to remove these and use the dvsim.py tool instead.

Reply:
These are removed in the amend. I can run dvsim.py for regression